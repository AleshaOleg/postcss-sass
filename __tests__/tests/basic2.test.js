var getPostCssTreeFromSass = require('../../helpers/getPostCssTreeFromSassTree');

it('basic2.sass', function () {
    expect(getPostCssTreeFromSass('basic2')).toMatchSnapshot();
});
