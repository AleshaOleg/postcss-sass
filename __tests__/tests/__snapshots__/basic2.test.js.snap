// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`basic2.sass 1`] = `
Object {
  "nodes": Array [
    Object {
      "nodes": Array [
        Object {
          "prop": "background",
          "raws": Object {
            "before": "
    ",
            "between": ": ",
            "semicolon": false,
          },
          "source": Object {
            "end": Object {
              "column": 20,
              "line": 2,
            },
            "input": Input {
              "css": "body
    background: #000

.one
    background: #000

    .two
        background: #f00

    .three
        background: #f00
",
              "id": "<input css 1>",
            },
            "start": Object {
              "column": 5,
              "line": 2,
            },
          },
          "type": "decl",
          "value": "#000",
        },
      ],
      "raws": Object {
        "before": "",
        "between": "",
      },
      "selector": "body",
      "source": Object {
        "end": Object {
          "column": 21,
          "line": 2,
        },
        "input": Input {
          "css": "body
    background: #000

.one
    background: #000

    .two
        background: #f00

    .three
        background: #f00
",
          "id": "<input css 1>",
        },
        "start": Object {
          "column": 1,
          "line": 1,
        },
      },
      "type": "rule",
    },
    Object {
      "nodes": Array [
        Object {
          "prop": "background",
          "raws": Object {
            "before": "
    ",
            "between": ": ",
            "semicolon": false,
          },
          "source": Object {
            "end": Object {
              "column": 20,
              "line": 5,
            },
            "input": Input {
              "css": "body
    background: #000

.one
    background: #000

    .two
        background: #f00

    .three
        background: #f00
",
              "id": "<input css 1>",
            },
            "start": Object {
              "column": 5,
              "line": 5,
            },
          },
          "type": "decl",
          "value": "#000",
        },
        Object {
          "nodes": Array [
            Object {
              "prop": "background",
              "raws": Object {
                "before": "
        ",
                "between": ": ",
                "semicolon": false,
              },
              "source": Object {
                "end": Object {
                  "column": 24,
                  "line": 8,
                },
                "input": Input {
                  "css": "body
    background: #000

.one
    background: #000

    .two
        background: #f00

    .three
        background: #f00
",
                  "id": "<input css 1>",
                },
                "start": Object {
                  "column": 9,
                  "line": 8,
                },
              },
              "type": "decl",
              "value": "#f00",
            },
          ],
          "raws": Object {
            "before": "

    ",
            "between": "",
          },
          "selector": ".two",
          "source": Object {
            "end": Object {
              "column": 25,
              "line": 8,
            },
            "input": Input {
              "css": "body
    background: #000

.one
    background: #000

    .two
        background: #f00

    .three
        background: #f00
",
              "id": "<input css 1>",
            },
            "start": Object {
              "column": 5,
              "line": 7,
            },
          },
          "type": "rule",
        },
        Object {
          "nodes": Array [
            Object {
              "prop": "background",
              "raws": Object {
                "before": "
        ",
                "between": ": ",
                "semicolon": false,
              },
              "source": Object {
                "end": Object {
                  "column": 24,
                  "line": 11,
                },
                "input": Input {
                  "css": "body
    background: #000

.one
    background: #000

    .two
        background: #f00

    .three
        background: #f00
",
                  "id": "<input css 1>",
                },
                "start": Object {
                  "column": 9,
                  "line": 11,
                },
              },
              "type": "decl",
              "value": "#f00",
            },
          ],
          "raws": Object {
            "before": "

    ",
            "between": "",
          },
          "selector": ".three",
          "source": Object {
            "end": Object {
              "column": 25,
              "line": 11,
            },
            "input": Input {
              "css": "body
    background: #000

.one
    background: #000

    .two
        background: #f00

    .three
        background: #f00
",
              "id": "<input css 1>",
            },
            "start": Object {
              "column": 5,
              "line": 10,
            },
          },
          "type": "rule",
        },
      ],
      "raws": Object {
        "before": "

",
        "between": "",
      },
      "selector": ".one",
      "source": Object {
        "end": Object {
          "column": 25,
          "line": 11,
        },
        "input": Input {
          "css": "body
    background: #000

.one
    background: #000

    .two
        background: #f00

    .three
        background: #f00
",
          "id": "<input css 1>",
        },
        "start": Object {
          "column": 1,
          "line": 4,
        },
      },
      "type": "rule",
    },
  ],
  "raws": Object {
    "before": "",
    "semicolon": undefined,
  },
  "source": Object {
    "end": Object {
      "column": 25,
      "line": 11,
    },
    "input": Input {
      "css": "body
    background: #000

.one
    background: #000

    .two
        background: #f00

    .three
        background: #f00
",
      "id": "<input css 1>",
    },
    "start": Object {
      "column": 1,
      "line": 1,
    },
  },
  "type": "root",
}
`;
